examples/lighting-app/esp32/sdkconfig.optimize.defaults

```diff
@@ -0,0 +1,93 @@
+#
```

Comment:
What does this have to do with the Python controller?

Reply:
yes its unrelated, mistakenly pulled latest changes from master  via clicking update branch button on github..

Reply:
This file still shows up.